Gruntfile.coffee

```diff
@@ -1240,6 +1240,16 @@ module.exports = (grunt) ->
 					"theme/**/*.js"
 					"tasks/*.js"
 				]
+			travis:
+				options:
+					configFile: '.travis.eslintrc.json'
+					quiet: true
+				all:
+					src: [
+						"src/**/*.js"
+						"theme/**/*.js"
+						"tasks/*.js"
+					]
```

Comment:
The issue is that the eslint has more than rules so my approach doesn't work. I wonder if you use both an eslintrc and a rules directive if it merges the two